ddtrace/profiling/collector/stack.pyx

```diff
@@ -296,14 +292,12 @@ cdef collect_threads(thread_id_ignore_list, thread_time, thread_span_links) with
 
 
 cdef stack_collect(ignore_profiler, thread_time, max_nframes, interval, wall_time, thread_span_links, collect_endpoint):
-
-    if ignore_profiler:
-        # Do not use `threading.enumerate` to not mess with locking (gevent!)
-        thread_id_ignore_list = {thread_id
-                                 for thread_id, thread in threading._active.items()
-                                 if getattr(thread, "_ddtrace_profiling_ignore", False)}
-    else:
-        thread_id_ignore_list = set()
+    # Do not use `threading.enumerate` to not mess with locking (gevent!)
```

Comment:
is this comment still valid? if we have our own copy of the threading module then it shouldn't matter?

Reply:
This sounds about right, but I haven't tested it. I'd be happy to make this a follow-up task. WDYT?